lightwood/helpers/device.py

```diff
@@ -36,3 +36,11 @@ def get_devices():
         available_devices = 0
 
     return torch.device(device_str), available_devices
+
+
+def get_device_from_name(device_name=''):
```

Comment:
I think you should modify all modified docstrings a bit.

Change all instances of:

> "name of the device to use (default is an empty string), if is an empty string will use the output of `get_devices()` instead") .

To:

> "name of the device that `get_device_from_name` will attempt to use.

And make the former to be the docstring in this method.